portal-api/src/v1/helpers/convertToTimestamp.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {String} time
+ * @returns {Date} timestamp
+ * Helper function to take time as String (eg July 19, 2022)
+ * Converts to utc timestamp by getting year date and month and setting hours to midnight
+ * returns in date format
+ */
+const convertToTimestamp = (time) => {
+  const date = new Date(time);
```

Comment:
Can we please add a check for `time` if the argument supplied is empty.

Reply:
done